docs/common/help.go

```diff
@@ -21,6 +21,19 @@ const GlobalEnvVars string = `	JFROG_CLI_LOG_LEVEL
 		[Default: The operating system's temp directory]
 		Defines the temp directory used by JFrog CLI.
 
+	JFROG_CLI_BUILD_NAME
+		Build name to use in build related commands. This environment variable will be used in case of omitted --build-name flag and <build name> argument.
+	
+	JFROG_CLI_BUILD_NUMBER
+		Build number to use in build related commands. This environment variable will be used in case of omitted '--build-number' flag and '<build number>' argument.
+
+	JFROG_CLI_BUILD_URL
+		Can be used for setting the CI server build URL in the build-info. This environment variable will be used in case of omitted '--build-url' flag.
```

Comment:
I suggest rephrasing to:
Sets the CI server build URL in the build-info. The "jfrog rt build-publish" command uses the value of this environment variable, unless the --build-url command option is sent.